sdk/storage/blobs/samples/src/blobs_client_example.c

```diff
@@ -86,6 +104,11 @@ az_storage_blobs_blob_delete(az_storage_blobs_blob_client* client, az_http_respo
 
 int main()
 {
+  /* Uncomment this to enable logging all http responses
```

Comment:
Put // on this line and then put /* on the next line.
As is, anyone who uncomments this will force a compiler error